src/pages/javascript/tutorials/how-to-create-a-dropdown-menu/index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: How to Create a Dropdown Menu with CSS and JavaScript
+title: How to Create a Dropdown Menu
```

Comment:
I think you can remove this article instead of renaming it, it's not relevant to this section.

Reply:
There are a bunch of how-to articles like this. Should I remove them all?

Reply:
@dhanushuUzumaki Maybe, can you list some of the ones that don't belong? Maybe we should look at moving them to the correct place instead. That's out of the scope of this PR though, we can do that in a different one.

Reply:
Sure. I will just address these comments and push to this one and create a new issue to move the how-to articles.